features/private_datasets.feature

```diff
@@ -44,7 +44,7 @@ Feature: Private Dataset API
             forbidden - dataset already exists
             """
 
-    Scenario: Adding canonical and subtopic fields to a dataset
+    Scenario: Adding topic and survey fields to a dataset
```

Comment:
I wonder why we are adding survey to the topic scenario. Maybe split them or even better have a scenario with all the properties set?